site/content/en/docs/Tutorials/external_tutorials.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "External Tutorials On Minikube"
+linkTitle: "External Tutorials On Minikube"
+weight: 1
+date: 2019-10-31
+description: >
+  Adding link to external tutorials based on minikube
+---
+
+Some of the good blogs posts and tutorials that are based on minikube are given below.
+
+Read more about minikube here: https://quarkus.io/guides/kubernetes-guide.html#deploying-the-application-in-kubernetes
```

Comment:
Obviously this place is now where to "read more about minikube", it is about Quarkus (a Java stack) ?

And it seems they moved the page, it is now here: https://quarkus.io/guides/deploying-to-kubernetes

Reply:
Yes they have moved the page here https://quarkus.io/guides/deploying-to-kubernetes

Reply:
I will update asap